src/components/chat-view-container/chat-view-container.tsx

```diff
@@ -151,6 +156,14 @@ export class Container extends React.Component<Properties> {
     }
   };
 
+  handleHiddenMessageInfoClick = () => {
+    if (!this.props.backupExists) {
+      this.props.openCreateBackupDialog();
+    } else {
+      this.props.openRestoreBackupDialog();
+    }
+  };
+
```

Comment:
Maybe this has always been a bug, but if you have a backup and it's verified, clicking this will tell you that your backup isn't verified.

We can have situations where we have hidden messages that are out of the users control
![2025-04-09 07 28 55](https://github.com/user-attachments/assets/8aa10ce6-5294-4df5-947e-fcd701a2f09e)

Reply:
This actually might be something weird going on with my browser. I can't get my backup to verify in brave, but it works in chrome. If you can't reproduce feel free to ignore.
I'll keep digging into what the underlying issue is on my end

Reply:
Yeah some messages will remain hidden, for example if the user receives a message while they are logged out, I'll add something to indicate this in the UI as I don't think we are at the moment which could be confusing if the user has restored their backup and still sees the message hidden.

As for the issue, I can also reproduce this, locally on this branch, main branch & on production. Also looking into it